src/test/java/org/broadinstitute/hellbender/utils/read/GATKReadAdaptersUnitTest.java

```diff
@@ -24,9 +27,12 @@
     public static final byte[] BASIC_READ_BASE_QUALITIES = {30, 40, 30, 50};
     public static final String BASIC_READ_CIGAR = "1M1I2M";
     public static final String BASIC_READ_GROUP = "FOO";
+    public static final String BASIC_PROGRAM = "x";
     public static final int BASIC_READ_END = BASIC_READ_START + TextCigarCodec.decode(BASIC_READ_CIGAR).getReferenceLength() - 1;
     public static final String BASIC_READ_MATE_CONTIG = "1";
     public static final int BASIC_READ_MATE_START = 125;
+    public static final String READ_GROUP_TAG = "RG"; // SAMTag.RG.name()
```

Comment:
If you want to alias these it shouldn't it be  `READ_GROUP_TAG = SAMTag.RG.name();` instead of redefining the constant?  It's super unlikely that `RG` will ever be changed, but I'm not sure why you wouldn't just set it explicitly to the sam tag value?